lib/connection/pool.js

```diff
@@ -783,10 +790,9 @@ Pool.prototype.connect = function() {
  * @param {...object} param Parameters for the specific mechanism
  * @param {authResultCallback} callback A callback function
  */
-Pool.prototype.auth = function(mechanism) {
+Pool.prototype.auth = function(credentials, callback) {
```

Comment:
Love the consistency of the `credentials` refactor! Please document the new parameter though